robottelo/ui/hosts.py

```diff
@@ -194,6 +195,12 @@ def update_host_bulkactions(
                         parameter['command']
                     )
                     self.click(common_locators['submit'])
+                if action == 'Delete Hosts':
+                    self.click(locators['host.bulk_submit'])
+                    return self.wait_until_element(
+                        locators['task.finished_state'],
+                        timeout=parameter['timeout'],
```

Comment:
perhaps we should use a default value like so we can omit every time putting the timeout value: timeout=parameter.get('timeout' , 300)
what do you think?